mathics/core/builtin.py

```diff
@@ -462,12 +462,17 @@ def get_functions(self, prefix="eval", is_pymodule=False):
                 if pattern is None:  # Fixes PyPy bug
                     continue
                 else:
-                    # TODO: consider to use a more sophisticated
+                    # TODO 1: consider to use a more sophisticated
```

Comment:
I suspect regular expressions aren't going to be the solution. 

And this kind of stuff (not matching package context) might be why we've been getting failures in running packages.